osi_common.proto

```diff
@@ -164,10 +175,21 @@ message MountingPosition
 //
 // Used e.g., for low level representations of radar detections.
 // 
-// Units are [m] for radial distance and [rad] for azimuth and elevation angles. 
+// Units are [m] for radial distance and [rad] for azimuth and elevation angles.
+// With the x-axis of the sensor frame pointing in the central viewing direction 
```

Comment:
We may want to rephrase this definition to be consistent with the defintion of the sensor frame in OSI:DetectionHeader. I propose a reference to avoid redundancies:
``Azimuth and elevation are defined as the rotations that would have to be applied to the sensor frame (definition in OSI:DetectionHeader) to make its x-axis point towards the referenced point or to align it with the referenced vector. The rotations are to be performed \b azimuth \b first (around the z-axis) and \b elevation \b second (around the new y-axis) to follow the definition of OSI:Orientation3D. For the sense of each rotation, the right-hand rule applies.``
I am not sure, however, whether OSI:DetectionHeader is the optimal location for the original definition.

Reply:
Normally detections uses spherical coordinates but it should be a common definition. I adapted your definition.